public/resources/casestudies/hdi/asyncapi.yaml

```diff
@@ -0,0 +1,151 @@
+asyncapi: 3.0.0
+info:
+  title: customer-example
+  version: 1.0.0
+  description: |
+    This is an AsyncAPI document for customer-example.
+    It contains every Topic owned by the customer in form of channel. 
+    Dowload the coresponding schema files from the following link:
+servers:
+  box-apac:
+    host: namespace.servicebus.windows.net
+    protocol: amqp
+    description: Azure Service Bus namespace endpoint for box.
+  box-emea:
+    host: namespace.servicebus.windows.net
+    protocol: amqp
+    description: Azure Service Bus namespace endpoint for box.
+channels:
+  helloworld-emea:
+    servers:
+      - $ref: '#/servers/box-emea'
+    address: https://namespace.servicebus.windows.net/topic/example/helloworld
+    messages:
+      helloworld:
+        name: helloworld-Message
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/helloworld-topic-v1.json'
+  test-topic-emea:
+    servers:
+      - $ref: '#/servers/box-emea'
+    address: https://namespace.servicebus.windows.net/topic/example/test-topic
+    messages:
+      test-topic:
+        name: test-topic-Message
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/test-topic-v1.json'
+  test-pipeline-emea:
+    servers:
+      - $ref: '#/servers/box-emea'
+    address: https://namespace.servicebus.windows.net/topic/example/test-pipeline
+    messages:
+      test-pipeline:
+        name: test-pipeline-Message
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/test-topic-v1.json'
+  source1-emea:
+    servers:
+      - $ref: '#/servers/box-emea'
+    address: https://namespace.servicebus.windows.net/topic/example/source1
+    messages:
+      source1:
+        name: source1-Message
+        contentType: application/json
+        payload:
+          $ref: '#/components/schemas/test-topic-v1.json'
+operations:
```

Comment:
this example shows an app that is a producer of events

is it coincidence or is it that your flow is to only use AsyncAPI to describe producers, and you never use "receive" operation?

Reply:
It was a deliberate decision to only use AsyncApi to to describe producers and by doing that create a catalogue of topics that can be accessed by potential subscribers